eZ/Publish/Core/settings/repository/inner.yml

```
@@ -9,6 +9,7 @@ services:
         arguments:
             - "%ezpublish.api.inner_repository.class%"
             - "%ezpublish.api.role.policy_map%"
+            - '@eZ\Publish\Core\Repository\SiteAccessAware\Language\LanguageResolver'
```

Let the API decide what to inject:
```suggestion
            - '@eZ\Publish\API\Repository\LanguageResolver'
```